### PR TITLE
Apply common style.

### LINKED ISSUE #79: Use common clang-format
Replace current clang-format configuration with https://github.com/prologin/prologin2019/blob/master/.clang-format and reformat all the codebase.

### PR DESCRIPTION
**Pull request by audebert**
Fixes #79

### CODE REVIEW

File: games/plusminus/champions/cxx/prologin.hh [hunk 1]
```diff
@@ -82,15 +82,16 @@ struct hash<std::vector<T>>
...
 } // namespace std
 
-extern "C" {
+extern "C"
+{
 
-/// Function called at the start of the game
-void init_game();
+    /// Function called at the start of the game
+    void init_game();
```
Comment (Shaac) [inline games/plusminus/champions/cxx/prologin.hh:91]: Maybe we want to add an option to not indent extern C. The same way we don’t indent namespaces.

File: tools/rubygenerator/files/rules.cc [hunk 2]
```diff
@@ -8,7 +8,7 @@
 #include "actions.hh"
 
 Rules::Rules(const rules::Options opt)
-    :!!rules_cc_inheritance!! sandbox_(opt.time)
+    : !!rules_cc_inheritance !!sandbox_(opt.time)
```
Comment (Shaac) [inline tools/rubygenerator/files/rules.cc]: This is a ~template. This will probably break the ruby generator.

Comment (audebert) [reply tools/rubygenerator/files/rules.cc]: Nice catch!

File: games/plusminus/champions/cxx/prologin.hh (hunk 1 above)
Comment (audebert) [reply games/plusminus/champions/cxx/prologin.hh:91]: Done.

**Pull request merged**